src/groups/mqb/mqbblp/mqbblp_relayqueueengine.cpp

```diff
@@ -80,6 +80,10 @@ const bsls::Types::Int64 k_NS_PER_MESSAGE =
     BALL_LOGTHROTTLE_INFO(k_MAX_INSTANT_MESSAGES, k_NS_PER_MESSAGE)           \
         << "[THROTTLED] "
 
+#define BMQ_LOGTHROTTLE_WARN()                                                \
```

Comment:
```suggestion
#define BMQ_LOGTHROTTLE_WARN                                                  \
```

The similar simplification can be done here

Reply:
amended